torch_xla/csrc/ir.cpp

```diff
@@ -153,11 +165,11 @@ Node::Node(torch::lazy::OpKind op, OpList operands,
 
 Node::Node(torch::lazy::OpKind op, xla::Shape shape, size_t num_outputs,
            torch::lazy::hash_t hash_seed)
-    : op_(std::move(op)),
-      num_outputs_(num_outputs),
-      shape_(std::move(shape)),
-      node_hash_(GetOpHash(op_, shape_, hash_seed)),
-      hash_(node_hash_) {
+    : torch::lazy::Node(op, num_outputs, /* hash_func */
+                        [&](bool /*bakeInSizes*/) -> torch::lazy::hash_t {
+                          return GetOpHash(op, shape, hash_seed);
```

Comment:
OK it seem like you already ignore `bakeInSizes` here, you should also update other Node constructors to do the same.